packages/sitecore-jss-nextjs/src/services/component-props-service.ts

```diff
@@ -123,6 +123,11 @@ export class ComponentPropsService {
    * Go through layout service data, check all renderings using displayName, which should make some side effects.
    * Write result in requests variable
    * @param {Object} params params
+   * @param {PlaceholdersData} params.placeholders
+   * @param {FetchFunctionFactory<NextContext>} params.fetchFunctionFactory
+   * @param {LayoutServiceData} params.layoutData
+   * @param {NextContext} params.context
+   * @param {ComponentPropsRequest<NextContext>[]} params.requests
```

Comment:
Same here - not sure if these are necessary?
```suggestion
```

Reply:
@ambrauer Here is necessary, we need to specify properties of object, if we can